packages/easy-coding-standard/README.md

```diff
@@ -38,18 +38,23 @@ use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigura
 use Symplify\EasyCodingStandard\ValueObject\Set\SetList;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
-    // A. standalone rule
+
+    // A. full sets
+    $containerConfigurator->import(SetList::PSR_12);
+
+    // B. standalone rule
     $services = $containerConfigurator->services();
     $services->set(ArraySyntaxFixer::class)
         ->call('configure', [[
             'syntax' => 'short',
         ]]);
-
-    // B. full sets
-    $containerConfigurator->import(SetList::PSR_12);
 };
 ```
 
+#### Important
```

Comment:
The generic words ("warning", "imporant", "do not miss this title") could be easily ignored, as they are overused. It's better to use exact titles that refer to specifically content bellow. 

How can we write it better?

Reply:
Configuration recommendations?

Reply:
Still generic. How would you sum the paragraph it up in 3 words?

Reply:
hmm. 

Avoid configuration overrides
Ensure configurations respected
Full sets before standalone rules (a little more than 3 words) 
A over B. (to lure them to read?)

Reply:
> Full sets before standalone rules (a little more than 3 words)

👍